### ISSUE TITLE
Change the validation splitting strategy in the encoder

### ISSUE THREAD

**Issue by vrodriguezf** [pacmel/timecluster_hub#19]
In GitLab by @vrodriguezf on Oct 24, 2020, 16:57

Right now, the validation set used during the encoder training is just a random 20% split of the input artifact, made automatically by Keras using the argument `val_pct`. This implies that the validation data is used for the normalization of the training data, and this is a bad practice. The validation data must be normalized independently, using the means/stds of the training data.

The easiest way to solve this problem is to use a separate artifact as validation set, instead of splitting the one used for training. As an example, we can use days 1-10 for training. days 11-13 for validation, and days 14-15 for test. Note that, in the call to `model.fit`, the argument `validation_split` is no longer applicable, and we have to use the argument `validation_data` (See [docs](https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit))
